sdk-setup/src/mb2

```diff
@@ -298,13 +298,6 @@ GLOBAL OPTIONS
         regular use - create a script file named after the .spec file but with
         '.changes.run' extension to instruct $ME to generate changelog with
         'git-change-log <args>' instead.
-
-
-KNOWN ISSUES
-
-Build target snapshot is not reset automatically when the original target is
-changed just by removing packages. Package installation or update is needed to
-trigger the automatic reset.
```

Comment:
The solution does not cover plain rpm usage, so the note should be only updated, not fully removed.

Reply:
Updated note, now reads:
> Build target snapshot is not reset automatically when the original target is
changed just by removing packages via rpm(only zypper keeps a log of removals).
> Either use zypper for all package operations or a package installation or
update is needed to trigger the automatic reset.

Reply:
I would suggest: "[...] just by removing packages via plain RPM usage. Either use zypper for all package operations or use 'mb2 installdeps --reset' to ensure a snapshot is reset.

Reply:
Done